### PR TITLE
Issue 167 discrete si

### LINKED ISSUE #167: Further improve discretization of the serial interval
While writing the introduction, Robin decided that we should be using a more sophisticated method to calculate discrete serial intervals. This method is described in Cori et al, A New Framework and Software to Estimate Time-Varying Reproduction Numbers During Epidemics, 2013, Appendix 11.

### PR DESCRIPTION
**Pull request by rcw5890**
Closes #167

### CODE REVIEW

File: branchpro/data_library/serial_interval/parse_data_si.py [hunk 1]
```diff
@@ -52,8 +59,21 @@ def write_ser_int_data(name):
...
-        disc_w = [w_dist.cdf(s+0.5) - w_dist.cdf(s-0.5)
-                  for s in np.arange(1, 61)]
+
+        # Get the density weighted by the delay
+        def weighted_density(u):
+            return u * w_dist.pdf(u)
+
+        # Calculate discrete serial interval terms, for k = 1,2,3,...
+        disc_w = [
+            (1 + k) * w_dist.cdf(k + 1)
```
Comment (DavAug) [inline branchpro/data_library/serial_interval/parse_data_si.py:69]: Could you explain your strategy of deriving the discrete intervals in the docstring?

**Review by DavAug — CHANGES_REQUESTED** [review]
Thanks @rcw5890 ! I just have an annoying comment about documentation. Could you explain the method you use to derive the serial intervals in the docstring for future reference?

File: branchpro/data_library/serial_interval/parse_data_si.py (hunk 1 above)
Comment (rcw5890) [reply branchpro/data_library/serial_interval/parse_data_si.py:69]: Agreed, #176

**Review by DavAug — APPROVED** [review]
Thanks @rcw5890 ! Looks great!

**Pull request merged**